scripts/extern_defined.sh

```diff
@@ -39,6 +39,8 @@ PG_BIN=$(pg_config --bindir)/postgres
 nm -D  $PG_BIN | grep " T " | awk '{print $3}' | sed -e 's/@.*$//p'
 # global bss symbol in postgres
 nm -D  $PG_BIN | grep " B " | awk '{print $3}' | sed -e 's/@.*$//p'
+# postgres Initialized data (bbs), global symbols
+nm -D  $PG_BIN | grep " D " | awk '{print $3}' | sed -e 's/@.*$//p'
```

Comment:
Just curious, were undefined symbols silently passing compilation and that is how you noticed/added this?

Reply:
They were failing from CI [here](https://github.com/lanterndata/lantern/actions/runs/6290682651/job/17078321222#step:3:634)